chapter2/tests/t3/chapter2-t3-velero-eks-diff.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# Task 3 - find namespaces without the backup
+set -eo pipefail
+
+
+readonly SAVED_NS_URL='https://gist.github.com/dmitry-mightydevops/016139747b6cefdc94160607f95ede74/raw/velero.yaml'
```

Comment:
1. Tried to pass not existing file here, it printed output with contents of the 2d file (not user friendly behavior)

Reply:
Can you clarify under which circumstances it prints the file contents? I was unable to reproduce it myself.

Reply:
This is still doesn't work as expected, expected that your script will raise error if one of files or both doesn't exist on current url. 

1. I check it by adding not existing filename and your script returns list of namespaces from the 2nd file:
![image](https://user-images.githubusercontent.com/26448672/231623431-e6c88896-a212-42fe-9198-3fb8076b0374.png)

2. When we change name to some not existing for 2nd file, we have not clear and user friendly error
![image](https://user-images.githubusercontent.com/26448672/231623647-5781d900-54b6-404c-9eb0-0e3a25d4c66a.png)

Reply:
> This is still doesn't work as expected, expected that your script will raise error if one of files or both doesn't exist on current url.

Oh, yes, fixed